docs/root/operations/admin.rst

```diff
@@ -103,6 +103,15 @@ modify different aspects of the server:
 
     */failed_outlier_check*: The host has failed an outlier detection check.
 
+.. _operations_admin_interface_config_dump:
+
+.. http:get:: /config_dump
+
+  Dump currently loaded configuration from various Envoy components as JSON-serialized proto
+  messages. Currently, only route configs are available but more are on the way. See
+  envoy/admin/v2/config_dump.proto for more information.  That proto is in draft state and is
```

Comment:
Prefer direct link. We use `:repo:` for main Envoy repo, I wonder if we can have something equivalent for this repo.

Reply:
Yes easy to setup: https://github.com/envoyproxy/data-plane-api/blob/master/docs/conf.py#L41